tests/test_recursive.py

```diff
@@ -55,10 +57,49 @@ class Meta:
 class TestRecursiveField:
     @staticmethod
     def serialize(serializer_class, value):
+        def _recursively_populate_nullable_field(serializer, instance_type):
```

Comment:
🍷🧑‍🍳 This just runs through Nullable fields and sets them to `Null` explicitly. This is needed because some of the recent updates to DRF (as in, between the last tested version here and 3.15) have made handling nullable fields more consistent and the tests were failing where these weren't passed in but they appear in the `.data` property.